config/sync/block.block.pagetitle.yml

```diff
@@ -23,6 +23,7 @@ visibility:
     bundles:
       application: application
       article: article
+      recipe: recipe
```

Comment:
Are we still using this? Not a problem if we are, I got the impression from past discussions this was causing some headaches and was being removed in favour of a `$title` variable to be rendered in a given template.